specs/app.spec.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs'),
```

Comment:
it will search for this `specs/data/transactions/may/transactions.csv` in the root directory and when it will not find a file it's name `specs/data/transactions/may/transactions.csv` it will throw an error